README.md

```diff
@@ -1459,6 +1460,24 @@ def finish_guided_session
   [foreman](https://github.com/ddollar/foreman) to manage them.
 <sup>[[link](#foreman)]</sup>
 
+## Auto loader rules
+
+Prefer nesting a class within a module instead of using module as part of class name:
```

Comment:
The same is already described in ruby-styleguide

https://github.com/simpliflow/Ruby-Styleguide#namespace-definition